tcms/testruns/static/testruns/js/get.js

```diff
@@ -22,38 +22,28 @@ $(document).ready(() => {
     });
 
     $('.add-hyperlink-bulk').click(() => {
-        const checkboxes = selectedCheckboxes()
-        if (!checkboxes.length) {
+        const selected = selectedCheckboxes()
+        if (selected === {}) {
             return false
         }
 
-        const testExecutionIds = []
-        checkboxes.each((_index, checkbox) => {
-            const testExecutionId = $(checkbox).data('test-execution-id')
-            testExecutionIds.push(testExecutionId)
-        })
-
-        addLinkToExecutions(testExecutionIds)
+        addLinkToExecutions(selected.executionIds)
     })
 
     $('.remove-case-bulk').click(() => {
-        const checkboxes = selectedCheckboxes()
-        if (!checkboxes.length) {
+        const selected = selectedCheckboxes()
+        if (selected === {}) {
             return false
         }
```

Comment:
Same thing here. After I see the No executions selected warning I then see the "Are you sure" confirmation.